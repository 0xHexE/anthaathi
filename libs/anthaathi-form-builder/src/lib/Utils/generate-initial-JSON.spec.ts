import generateInitialJSON from './generate-initial-JSON';

describe('Generate initial JSON', () => {
  it('should render the items', () => {
    const data = generateInitialJSON({
      type: 'object',
      properties: { kind: { type: 'string' }, abs: { type: 'object' } },
    });

    expect(data).toMatchSnapshot();
  });
});
