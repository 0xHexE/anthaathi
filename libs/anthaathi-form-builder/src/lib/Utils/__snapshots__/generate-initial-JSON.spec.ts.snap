// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Generate initial JSON should render the items 1`] = `
Object {
  "abs": Object {},
  "kind": "",
}
`;
